cmake/FindZeroMQ.cmake

```diff
@@ -63,5 +63,13 @@ else (ZeroMQ_LIBRARIES AND ZeroMQ_INCLUDE_DIRS)
   # show the ZeroMQ_INCLUDE_DIRS and ZeroMQ_LIBRARIES variables only in the advanced view
   mark_as_advanced(ZeroMQ_INCLUDE_DIRS ZeroMQ_LIBRARIES)
 
+  if(ZeroMQ_FOUND AND NOT TARGET libzmq)
+    add_library(libzmq UNKNOWN IMPORTED)
```

Comment:
Using variables like `ZeroMQ_INCLUDE_DIRS` or `ZeroMQ_LIBRARIES` is old-school cmake and it is discouraged. All modern codebases should use targets instead. 
So I slightly tweaked this Find module to also expose a target, using a name that matches conan's.